app/main.py

```diff
@@ -1,8 +1,51 @@
+from __future__ import annotations
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> int | float:
```

Comment:
use more descriptive type annotations

```suggestion
    def serve_cars(self, cars: list[Car]) -> int | float:
```